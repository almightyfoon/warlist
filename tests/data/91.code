c1501b_-0n8U9cf69Aakanaj9ce5dXbz__eS9b9g9g7j7kaj9bg89u8Maya5a6a89T9X