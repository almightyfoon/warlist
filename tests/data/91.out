https://conflictchamber.com/#c1501b_-0n8U9cf69Aakanaj9ce5dXbz__eS9b9g9g7j7kaj9bg89u8Maya5a6a89T9X

Cygnar Army - 79 / 75 points
[Theme] Heavy Metal

!!! This army must contain 75 points or fewer

[Haley 2] Major Victoria Haley [+25]
 - Firefly [8]
 - Hurricane [39]
 - Thorn [13]
 - Squire [0(5)]
Captain Arlan Strangewayes [0(4)]
Journeyman Warcaster [0(4)]
 - Firefly [8]
Lieutenant Allison Jakes [4]
 - Reliant [14]
Storm Strider [18]

Cygnar Army - 75 / 75 points

[Haley 3] Major Prime Victoria Haley [+25]
 - Charger [9]
 - Grenadier [9]
 - Grenadier [9]
Anastasia di Bray [3]
Gorman di Wulfe, Rogue Alchemist [4]
Journeyman Warcaster [4]
 - Charger [9]
Major Harrison Gibbs [4]
Ragman [4]
Lady Aiyana & Master Holt [8]
 - Captain Jonas Murdoch [4]
Stormblade Infantry [10]
 - Stormblade Infantry Officer & Standard [5]
 - Stormblade Infantry Storm Gunner (2) [4]
Trencher Infantry (min) [10]
 - Trencher Infantry Rifle Grenadier (2) [4]
